scripts/update/deploy.py

```diff
@@ -48,7 +48,6 @@ def update_assets(ctx):
     with ctx.lcd(settings.SRC_DIR):
         ctx.local("python2.7 manage.py nunjucks_precompile")
         ctx.local("python2.7 manage.py collectstatic --noinput")
-        ctx.local("LANG=en_US.UTF-8 python2.7 manage.py compress_assets")
```

Comment:
If we don't do this at compile time, does it happen at request time? Or does `collectstatic` now do the compress_assets part?